### DISCUSSION TITLE
[7.0RC2] Testing new directories layout

### DISCUSSION THREAD

**Discussion by jman-schief** [BlitterStudio/amiberry#1562] [Q&A] [answered]
Hello @midwan  :wave: here I am with some observations after testing the latest RC2.

I am on Debian/trixie and have installed the latest bookworm 7.0 RC2 package. Then I launch amiberry as follows:
```
$ export AMIBERRY_DATA_DIR=$AMIGAHOME/amiberry/data
$ export AMIBERRY_HOME_DIR=$AMIGAHOME
$ export AMIBERRY_CONFIG_DIR=$XDG_CONFIG_HOME/amiberry
$ amiberry -o write_logfile=yes
```

I observe that:
- my configuration is loaded (yay!)
- the directory `$HOME/.local/share/amiberry` is still created, I expect to not be created but I think it is ignored.
- the logfile is created where I configure it in from the GUI (as explained [here](https://github.com/BlitterStudio/amiberry/wiki/GUI:-Paths)). But at this [other link](https://github.com/BlitterStudio/amiberry/wiki/Amiberry-directories#directories-in-home) I read that it will be created in `$AMIBERRY_HOME_DIR/rp9/amiberry.log`. Is the latter info outdated?
- Mixed casing: the documentation indicates `$AMIBERRY_HOME_DIR/Amiberry` and `$XDG_DATA_HOME/amiberry`. Is there a reason for that? Maybe retro-compatibility? We penguins love lowercase directory names :-)

As always: thanks for Amiberry :tada:

**Comment by midwan** [comment, marked as answer]
Thanks for the feedback!

- I believe the directory under `.local/share/amiberry` is created by the `.deb` installer itself, but if you're not using it, it should be OK
- Thanks for pointing that out to me. The wiki needed one more empty line, to show that text correctly. Fixed now. :)
- The mixed case mentioned in the wiki is correct. I tried to follow some standards that modern Linux distros have on the Desktop, where they capitalize the Home directories (e.g. `Downloads`, `Documents`) while having config directories in lowercase (e.g. the stuff under `.config/`).

**Reply by jman-schief** [reply]
> The wiki needed one more empty line, to show that text correctly

LOL

> I believe the directory under .local/share/amiberry is created by the .deb installer itself, but if you're not using it, it should be OK

At least on Debian it seems that `~/.local/share/amiberry` is created when I execute amiberry, I reinstalled amiberry to verify that. The installer creates the directory tree in `/usr/share/amiberry`. I was curious to see why and looked at [this line](https://github.com/BlitterStudio/amiberry/blob/6505ca86314b5171d33cd2cba4df46bfeeaff3ca/src/osdep/amiberry.cpp#L3805) in the code. Even if I enable logging, that line is not printed in the logs. Why?

ok for the rest, thanks for clarifying!

**Reply by midwan** [reply]
Ah right, it's not in the logs, because we haven't reached the point of loading the `amiberry.conf` file, to check if logging should write to the logfile or not. :D
